test/gtest/bn_fwd_train.cpp

```diff
@@ -32,13 +32,22 @@
 // BiasDataType    : half
 // MeanVarDataType : float
 
-struct GPU_BN_CK_FWD_Train_Large_FP16
-    : BNFwdTrainTest<half_float::half, half_float::half, half_float::half, half_float::half, float>
+struct GPU_BNCKFWDTrainLarge2D_FP16 : BNFwdTrainTest<half_float::half,
```

Comment:
Just curious why some of these are multi-line and some are single-line. Have you ran clang-format?